.github/workflows/snakemake-ci.yml

```diff
@@ -28,25 +28,28 @@ jobs:
         with: 
           environment-file: recast_helloworld/environment.yml
 
-      - name: List installed environment dependencies 
-        run: | 
-          micromamba list
-
-      - name: Linting
-        uses: snakemake/snakemake-github-action@v1
-        with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
-
       - name: Testing
         uses: snakemake/snakemake-github-action@v1
         with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
+          directory: 'recast_helloworld'
+          snakefile: 'recast_helloworld/Snakefile'
           args: '--cores 1 --use-conda --conda-cleanup-pkgs cache'
 
       - name: Create container file
         uses: snakemake/snakemake-github-action@v1
         with:
-          snakefile: 'workflow/Snakefile'
+          snakefile: 'recast_helloworld/Snakefile'
           task: 'containerize'
+          directory: 'recast_helloworld'
+
+      - name: Uploading a snakemake logs
+        uses: actions/upload-artifact@v3
+        with: 
+          name: snakemake-log
+          path: .snakemake/log/
```

Comment:
```suggestion
          path: recast_helloworld/.snakemake/log/

```

Reply:
Running this locally in a Docker container seems that you want to specify the directory path too. The printed log statements like

```
Complete log: .snakemake/log/2023-08-14T174726.115983.snakemake.log
```

are relative to the run directory given with `--directory`.